cedar-java-hello-world/README.md

```diff
@@ -1,24 +1,38 @@
 # Cedar Java Hello World
 
 This repository contains a simple hello world program written in Java, using Cedar.
-It shows how to use of the Cedar Java API to evaluate a simple policy. You can look at the gradle files to see how to build CedarJava in your own Java applications.
+It shows how to use the Cedar Java API to evaluate a simple policy. You can look at the gradle files to see how to build CedarJava in your own Java applications.
 
 ## Usage
 
-The Java example works with the version of Cedar available on Maven, which (at the time of writing) is v2.3.3.
+The Java example works with the version of Cedar available on Maven, which (at the time of writing) is v3.1.2.
 
-To build, you'll need CedarJava and CedarJavaFFI. You will need to ensure that CedarJava is able to find the dynamic library of Cedar. To do that, you need to ensure the environment variable `CEDAR_JAVA_FFI_LIB` gives the location of your `cedar_java_ffi` shared library. Typically this can be done by running:
+### Build
+
+#### Building with the Maven release
+
+You can switch to the cedar-java uber jar in `build.gradle` so that the example runs using both the CedarJava and
+CedarJavaFFI libraries from the Maven release of cedar-java:3.1.2.
 
 ```shell
-bash config.sh
+# Change the build.gradle to use the uber jar
+sed -i '' "s/'com.cedarpolicy:cedar-java:3.1.2'/'com.cedarpolicy:cedar-java:3.1.2:uber'/" build.gradle
```

Comment:
Why not just make this change to the gradle file in this PR? That way the instructions can just be `./gradlew build`

Reply:
Agreed 🤝 

Didn't know if using the default jar instead of the uber jar was intentional, I'll make this change.